db/migrate/20240515100754_create_medicin.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMedicin < ActiveRecord::Migration[7.0]
+  def change
+    create_table :medicins do |t|
+      t.string :name, null: false
+      t.string :leaflet, optional: true
```

Comment:
what is the idea behing this?

Reply:
I thought that it would be a good idea to have a space to permit the leaflet to be stored, while not being mandatory, to maybe allow in the future to search secondary effects of drugs or how a drug should be taken

Reply:
got it